assets/scripts/auth/ui.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const app = require('../app-data');
+
+const signInSuccess = (data) => {
+  app.user = data.user;
+  console.log(app);
+  console.log("Sign in successful");
+};
+
+const signOutSuccess = () => {
+  app.user = null;
+  console.log(app);
+  console.log("You signed bro. Sweet!");
+};
+
+const success = (data) => {
+  console.log(data);
+};
+
+const failure = (error) => {
+  console.error(error);
+};
+
+module.exports = {
+  failure,
+  success,
+  signOutSuccess,
+  signInSuccess,
+  app,
```

Comment:
Instead of 
```js
app.user = data.user;
```

I think it would be helpful to store user information in an object on the app file, and update the export to include app & currentUser
```js
let currentUser = {
  id: undefined,
  token:''
};
```